### PR TITLE
Feature/1816 - Add Richmond Int'l (RIC)

### LINKED ISSUE #1816: Add KRIC - Richmond, VA
This issue adds KRIC - Richmond Intl.

### PR DESCRIPTION
**Pull request by MichaelDMcCracken**
Resolves #1816 

The purpose of this pull request is to add KRIC.

### CODE REVIEW

File: documentation/airport-guides/kric.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# KRIC - Richmond International Airport
+
+## About RIC
+Richmond International Airport (IATA: RIC, ICAO: KRIC, FAA LID: RIC) is a joint civil-military airport in Sandston, Virginia, United States, an unincorporated community (in Henrico County). The airport is about 7 miles (11 km) southeast of downtown Richmond, the capital of the Commonwealth of Virginia. Richmond International Airport is the busiest airport in central Virginia and the third-busiest in the state[citation needed] behind Washington D.C.'s two major airports, Washington Dulles and Washington National. RIC covers 2,500 acres (1,012 ha) of land.(Wikipedia)
```
Comment (Fechulo) [inline documentation/airport-guides/kric.md:4]: We can get rid of the [citation needed]

File: assets/airports/kric.json [hunk 2]
```diff
@@ -0,0 +1,9359 @@
...
+        "V157": ["LVL", "MANGE", "DALTO", "MATOX", "RIC", "SVILL", "TAPPA", "DUPEY", "COLIN", "PXT"],
+        "V260": ["FKN", "WAIKS", "HPW", "RIC", "FAK"]
+    },
+    "sids": {
+        "COLIN8": {
+            "icao": "COLIN8",
+            "name": "Colin Eight",
+            "rwy": {
+                "KRIC2": [["^_RW021DME", "A007+"], "_RIC05010"],
+                "KRIC16": [["^_RW161DME", "A007+"], "_RIC12107", "_RIC07410", "MODEL"],
```
Comment (Fechulo) [inline assets/airports/kric.json]: _RIC12107 Should be fly-over and at or above 4000ft

File: assets/airports/kric.json [hunk 3]
```diff
@@ -0,0 +1,9359 @@
...
+                ["_RW161DME", "ITTEM", "_RW201DME"],
+                ["ITTEM", "KALLI"]
+            ]
+        },
+        "LUCYL6": {
+            "icao": "LUCYL6",
+            "name": "Lucyl Six",
+            "rwy": {
+                "KRIC2": [["^_RW021DME", "A007+"], ["SJAYY", "A50+"], "KAMMI"],
+                "KRIC16": [["^_RW161DME", "A007+"], ["KEGME", "A50+"], "HAYYU"],
```
Comment (Fechulo) [inline assets/airports/kric.json]: Should be KEGME at or above 4000ft, thereafter BUNKK at or above 5000ft

File: assets/airports/kric.json [hunk 4]
```diff
@@ -0,0 +1,9359 @@
...
+    "stars": {
+       "DUCXS5": {
+            "icao": "DUCXS5",
+            "name": "Ducks Five",
+            "entryPoints": {
+                "KELCE": [["KELCE", "A110|S250"]],
+                "NEAVL": [["NEAVL", "A110|S250"], "PULDY"]
+            },
+            "body": ["THINA"],
+            "rwy": {
```
Comment (Fechulo) [inline assets/airports/kric.json]: Although the chart states that vectors should be expected prior to THINA when landing North, aircraft would be expected to continue to DUCXS if the vectors aren't given in time.

File: documentation/airport-guides/kric.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+## Airspace - Potomac Tracon (James River / JRV Area)
+The PCT JRV sector extends from the surface to 12,000ft in most areas. For the purposes of Richmond, only the FLTRK ("Flat Rock") and TAPPA sectors are available. These sectors extend roughly 35 nautical miles in each direction from the airport. Within these sectors, there are a number of Restricted Areas and Military Operations Areas that aircraft must avoid.
```
Comment (Fechulo) [inline documentation/airport-guides/kric.md:7]: Is there a reason the CSIDE and CSIDW sectors haven't been included?

File: assets/airports/kric.json [hunk 6]
```diff
@@ -0,0 +1,9359 @@
...
+        "V38": ["GVE", "EMMIE", "ROOKY", "RIC", "NEMAN", "HCM", "JUDLU", "TERKS", "CCV"],
+        "V155": ["RDU", "WIPER", "BEXGO", "LVL"],
+        "V157": ["LVL", "MANGE", "DALTO", "MATOX", "RIC", "SVILL", "TAPPA", "DUPEY", "COLIN", "PXT"],
+        "V260": ["FKN", "WAIKS", "HPW", "RIC", "FAK"]
+    },
+    "sids": {
+        "COLIN8": {
+            "icao": "COLIN8",
+            "name": "Colin Eight",
+            "rwy": {
```
Comment (Fechulo) [inline assets/airports/kric.json]: It appears that the definition of the fixes are slightly off to what the should be, I'm not sure if it's an issue with units during the calculation (km vs nm) and/or magnetic deviation issues. In the image below I've marked in green where the fixes are currently defined, and in red the approximate position of where they should be.
image

File: assets/airports/kric.json [hunk 7]
```diff
@@ -0,0 +1,9359 @@
...
+            "draw": [
+                ["_RW021DME", "SJAYY", "_RW341DME"],
+                ["SJAYY", "KAMMI", "LUCYL*", "HAYYU", "BUNKK", "KEGME"],
+                ["_RW161DME", "KEGME", "_RW201DME"]
+            ]
+        },
+        "READE7": {
+            "icao": "READE7",
+            "name": "Reade Seven",
+            "rwy": {
```
Comment (Fechulo) [inline assets/airports/kric.json]: Less noticeable than with the COLIN8, but there is still a problem with magnetic deviation by the looks of it
image

**Review by Fechulo — CHANGES_REQUESTED** [review]

File: documentation/airport-guides/kric.md (hunk 5 above)
Comment (MichaelDMcCracken) [reply documentation/airport-guides/kric.md:7]: Looking again at the SOPs, the primary reason is that in my (probably 50 or more) times flying into RIC in real life, I've never been handed off to those frequencies. While they are technically part of Potomac, but don't really serve RIC directly.

File: assets/airports/kric.json (hunk 3 above)
Comment (MichaelDMcCracken) [reply assets/airports/kric.json]: Can't believe I missed an entire fix 😅 fixed

File: documentation/airport-guides/kric.md (hunk 5 above)
Comment (Fechulo) [reply documentation/airport-guides/kric.md:7]: Good enough for me

**Pull request merged**